Tests/MeiliSearchUnitTests/SystemTests.swift

```diff
@@ -30,15 +30,67 @@ class SystemTests: XCTestCase {
 
         // Start the test with the mocked server
 
-        let expectation = XCTestExpectation(description: "Check server health")
+        let expectation = XCTestExpectation(description: "Check body of health server on health method")
 
         self.client.health { result in
             switch result {
             case .success(let body):
                 XCTAssertEqual(expectedHealthBody, body)
                 expectation.fulfill()
             case .failure:
-                XCTFail("Failed to check server health")
+                XCTFail("Failed on available status check on health method")
+            }
+        }
+
+        self.wait(for: [expectation], timeout: 1.0)
+
+    }
+
+    func testIsHealthyTrue() {
+
+        //Prepare the mock server
+
+        let jsonString = """
+        {
+            "status": "available"
+        }
+        """
+
+        session.pushData(jsonString, code: 200)
```

Comment:
```suggestion

        session.pushData("", code: 200)
```

Reply:
The test failed without the json. I think it's because the `isHealthy()`  method call `health()` method who need it.

Reply:
Oh yes probably! I forgot about that :)